homeassistant/helpers/icon.py

```diff
@@ -18,3 +18,14 @@ def icon_for_battery_level(
     elif 5 < battery_level < 95:
         icon += "-{}".format(int(round(battery_level / 10 - 0.01)) * 10)
     return icon
+
+
+def icon_for_signal_level(signal_level: Optional[int] = None) -> str:
```

Comment:
@balloob did you approve this?

Reply:
This is ok. We're using it in multiple places.